src/components/login_form.js

```diff
@@ -0,0 +1,101 @@
+import React, { Component, PropTypes } from 'react';
+import { reduxForm } from 'redux-form';
+import Link from 'react-router';
+
+import { createAuth, loginAuth } from '../actions/index';
+
+class LoginForm extends Component {
+  componentWillMount() {
+    //  create auth and put in props
+    this.props.createAuth();
+  }
+
+  static contextTypes = {
+    router: PropTypes.object
+  };
+
+  onSubmit(input) {
+    const { auth } = this.props;
+
+    this.props.loginAuth(input, auth);
+  }
+
+  redirect() {
+    const { auth, login } = this.props;
+
+    // save login in localStorage before redirection
+    localStorage.setItem('login', JSON.stringify(login));
+    // sets a cookie on redirect
+    auth.session.setCookieAndRedirect(login.res.sessionToken, auth.options.redirectUri + 'home'); 
+  }
+
+  render() {
+    const { fields:  { username, password }, login, handleSubmit } = this.props;
+
+    if(login.success) {
+      this.redirect();
+    }
+
+    return (
+      <div className="row under-header">
```

Comment:
In general, I'm not sure if we actually want the example to show a login form - I think the two main examples are:
1) Use the okta-signin-widget in a react app. We can customize the CSS, etc to show how flexible it is. One reason this is better than showing our own custom form is that it takes care of the other login states for us (expired password, recovery flows, warnings, etc) - I don't think we want them to go down that flow.
2) Use okta-auth-js for the OIDC flows (i.e. you get a popup that returns you an idToken/accessToken)